### DISCUSSION TITLE
Join

### DISCUSSION THREAD

**Discussion by fpsilva-source** [rod-trent/MustLearnKQL#13] [Q&A] [answered]
Hello,

How do I make the join between two tables with different fields between them?

**Comment by SQLtattoo** [comment, marked as answer]
Hello, there is the logic of the left and right table.
Let's say you have the following 2 tables: T1, T2

```
T1 //is the $left table 
| project procId, procName
| join kind=inner (
         T2 //is the $right table 
         | project _procId, _procName
) on $left.procID == $right._procId
```

So you can chain tables like that and use $left, $right in the sequence to reference the table and its column name.

Check the reference MS Doc here [https://docs.microsoft.com/en-us/azure/data-explorer/kusto/query/joinoperator?pivots=azuredataexplorer](https://docs.microsoft.com/en-us/azure/data-explorer/kusto/query/joinoperator?pivots=azuredataexplorer)

Hope this helps.
